browser-extension/addTournesolRecommendations.js

```diff
@@ -182,7 +182,9 @@ const getTournesolComponent = () => {
        video_score.className = 'video_text';
        video_score.append(
          'Rated ' +
-           Number(video.criteria_scores[0].score) +
+           Number(video.criteria_scores[video.criteria_scores.findIndex(element=>{
+           return  element.criteria==='largely_recommended'? true:false
+           })].score) +
```

Comment:
You don't need to use `findIndex` then retrieve this index from the array. The `find` method does it for you: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

`element.criteria==='largely_recommended'? true:false` is a beginner mistake, simply `element.criteria==='largely_recommended'` is exactly the same thing